### DISCUSSION TITLE
4A9C93EF-15F3-967B-8D0A-86BFE4E361DB

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#108] [General]
# 4A9C93EF-15F3-967B-8D0A-86BFE4E361DB

HD 45166 is a binary star system comprising of a B-type main-sequence star and a quasi-Wolf-Rayet star about 4,851 light-years away in the constellation Monoceros.

https://celestia.mobi/addon?item=4A9C93EF-15F3-967B-8D0A-86BFE4E361DB

**Comment by exocritic5457** [comment]
Will this get an update along with the addition of those pulsar systems?

**Reply by SevenSpheres** [reply]
It was [already updated](https://celestiaproject.space/forum/viewtopic.php?p=154680#p154680), but apparently not submitted to mobi yet
